prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -640,6 +644,7 @@ class End2End : CoolTest() {
     override val status = TestStatus.SMOKE
 
     override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        initListOfGoodReceiversAndCounties(environment)
         var passed = true
```

Comment:
I know this is not part of this PR, but why do we assume tests pass? Should this not be `false` and set explicitly to true in the else branch that starts on L667?

Reply:
Since any one failure should fail the whole test, the most important logic is to make sure you don't overwrite an early  failure with a success later in the test.   Setting it to true once only, at the start, and then all other sets are to false, is a way to minimize (but not eliminate) the chance of a developer messing that up.